components/issue-selector/issue-selector.jsx

```diff
@@ -1,15 +1,27 @@
 import React from 'react';
 import NavLink from '../nav-link/nav-link.jsx';
+import Utility from '../../js/utility';
+
+const ISSUES = [
```

Comment:
Given that we get the list of known issues from the `/issues` REST endpoint, including the description, does it make more sense to no longer hardcode these labels and instead add these labels the API server for each issue? (we can do that in follow-up though)